openmc/material.py

```diff
@@ -523,6 +523,95 @@ def remove_element(self, element):
             if element == elm:
                 self._elements.remove(elm)
 
+    def add_material(self, material, percent, percent_type='ao'):
+        """Add a material to the material
```

Comment:
I like the changing of the wording for the comment, but I think it is important to keep the method name consistent since it shares so many characteristics to the `add_element` and `add_nuclide` methods.